frontend/src/components/supp-allowances/SupportNeedsProgrammingAllowance.vue

```diff
@@ -25,16 +25,21 @@
   </v-row>
   <v-divider class="mt-2"></v-divider>
   <v-row no-gutters class="mr-2 my-2">
-    <v-col cols="12">Please describe how you intend to use this funding</v-col>
+    <v-col cols="12">Please describe how you intend to use this funding:</v-col>
   </v-row>
 
-  <v-row v-for="item in CHECKBOX_LABELS" :key="item.value">
-    <v-checkbox v-model="model.supportFundingModel" density="compact" class="pl-8" prepend-icon :label="item.label" :value="item.value"></v-checkbox>
-    <v-tooltip v-if="item.tooltip" content-class="tooltip" :text="item.tooltip">
-      <template #activator="{ props }">
-        <v-icon size="large" v-bind="props">mdi-information-slab-circle-outline</v-icon>
-      </template>
-    </v-tooltip>
+  <v-row v-for="item in CHECKBOX_LABELS" :key="item.value" no-gutters>
+    <v-col cols="11" lg="6">
+      <v-checkbox v-model="model.supportFundingModel" density="compact" class="pl-lg-8 mr-0" prepend-icon :label="item.label" :value="item.value"></v-checkbox>
+    </v-col>
+
+    <v-col cols="1">
```

Comment:
I think this one will look better and we don't need to rely on column since it looks different for each screen size:
```
<v-col cols="11" lg="6">
      <v-checkbox v-model="model.supportFundingModel" density="compact" class="pl-lg-8 mr-0" prepend-icon :value="item.value">
        <template v-slot:label>
          <p>
            {{ item.label }}
            <v-tooltip v-if="item.tooltip" content-class="tooltip" :text="item.tooltip">
              <template #activator="{ props }">
                <v-icon size="large" v-bind="props">mdi-information-slab-circle-outline</v-icon>
              </template>
            </v-tooltip>
          </p>
        </template>
      </v-checkbox>
    </v-col>
```

Reply:
done